oak-run-elastic/src/main/java/org/apache/jackrabbit/oak/index/ElasticIndexCommand.java

```diff
@@ -38,6 +39,7 @@
 import org.apache.jackrabbit.oak.run.cli.Options;
 import org.apache.jackrabbit.oak.run.commons.Command;
 import org.apache.jackrabbit.oak.run.commons.LoggingInitializer;
+import org.apache.jackrabbit.oak.segment.file.tar.index.Index;
```

Comment:
As we are anyway changing this... could you change the import org.apache.jackrabbit.oak.spi.commit.* (I would avoid star imports)

Reply:
Done